composer.json

```diff
@@ -51,6 +55,17 @@
         ]
     },
     "scripts": {
+        "dependency-cleanup": [
+            "rm -rf bin",
```

Comment:
Please add this cleanup to the script handler class instead of executing the rm directly in the composer json.